vnext/local-cli/src/runWindows/utils/deploy.ts

```diff
@@ -54,7 +59,16 @@ function getAppPackage(options: RunWindowsOptions): string {
   const appPackageGlob = `${
     options.root
   }/windows/{*/AppPackages,AppPackages/*}/${packageFolder}`;
-  let appPackage = glob.sync(appPackageGlob)[0];
+  const globs = glob.sync(appPackageGlob);
+  let appPackage;
+  if (globs.length == 1 || !projectName) {
+    appPackage = globs[0];
+  } else if (globs.length > 1) {
+    const filteredGlobs = globs.filter(x => x.indexOf(projectName) != -1);
```

Comment:
Nit: Use the "includes" function instead of comparing indexof to -1. Same below. 

Same comment above about terminology, where globs describe the pattern, which isn't quite accurate for the variable name here.